opdracht-1.js

```diff
@@ -0,0 +1,44 @@
+console.log("..............1A..............");
+// 1a
+const tvType = inventory.map((tv) => {
+    tv = tv.type
+    return tv
+});
+
+console.log(tvType);
+
+console.log("..............1B..............");
+// 1b
+const soldOut = inventory.filter((stock) => {
+    if(stock.originalStock - stock.sold === 0) {
+        return stock
+    }
+});
```

Comment:
De filter functie verwacht een boolean als resultaat in plaats van een object. Als het resultaat `true` is, dan komt de tv wél terug in de nieuwe array, bij `false` niet
```suggestion
const soldOutTvs = inventory.filter(tv => tv.originalStock === tv.sold);
```